src/library/cratefeature.cpp

```diff
@@ -222,9 +293,7 @@ void CrateFeature::onRightClickChild(const QPoint& globalPos, QModelIndex index)
     }
 
     bool locked = m_crateDao.isCrateLocked(crateId);
-
-    m_pDeleteCrateAction->setEnabled(!locked);
```

Comment:
I'll add more complex handling of this. We probably need to check if there is any children locked.